File: HW01/CPP/src/CampingTent.cpp
#include "../include/CampingTent.h"
#include <iostream>
#include <string>
using namespace std;

CampingTent::CampingTent(string size, int durability, string rating, double price) : OutdoorItem(price){
    this->size = size;
    this->durability = durability;
    this->rating = rating;
    this->price = price;
}

void CampingTent::display() {
    cout << "$" << this->price << " - This camping tent is size " << this->size << " and has a durability rated for " << this->durability << " years. It is also rated for "  << this->rating << " winds and rains" << endl;
}

File: HW01/PYTHON/project/CampingTent.py
from .OutdoorItem import OutdoorItem

class CampingTent(OutdoorItem):
    def __init__(self, price, size, durability, rating):
        super().__init__(price)
        self.size = size
        self.durability = durability
        self.rating = rating
        
    def display(self):
        print(f"${self.price} - This camping tent is size {self.size} and has a durability rated for {self.durability} years. It is also rated for {self.rating} winds and rains", end="")